### DISCUSSION TITLE
[Show and Tell] <ACryptoS - Liquidity Manager for V3 CL DEXs and Lending Protocols>

### DISCUSSION THREAD

**Discussion by CryptoBKT** [ethereum-optimism/developers#99] [Show and tell]
### [ACryptoS - Advanced Crypto Strategies](https://app.acryptos.com/)

ACryptoS is a liquidity manager and yield optimizer, and we've been building on OP Mainnet since November 2022. Over the past year, we've brought in TVL via our unique strategies that automate + increase APY for users.

Here's some of our key products on Optimism:

**1. Single-Token Vaults:**
- We automate a supply/borrow loop on lending protocols, leveraging the same-token, thus increasing yields for users, and increasing utilization for the lending protocol.
- Users are not exposed to liquidation risks as the supply/borrow loop uses the same token.
- These vaults have attracted strong TVL since launch, reaching a peak of $10+M early this year.
- We started by building with Sonne Finance, then expanded to partner with dForce Network.
Docs: https://docs.acryptos.com/products/vaults/single-token-vaults

**2. Automated Conc. Liquidity Manager (ACLM) Vaults:**
- We simplified the process for users when providing liquidity to V3 CL DEXs. Users can easily put funds in UniswapV3 LPs, without the hassle of setting positions/ranges, or constantly worry that the positions might go out of range
- Our strategies fully automate the ranges and positions, and rebalance whenever needed. Users can also deposit single tokens to the LP without needing to swap beforehand
- These ACLM vaults have been achieving good APY consistently, eg. 20-30%+ for stablecoin pairs, and 10%+ for wstETH-ETH
. Pairs like OP-ETH have been performing great as well. APY is fully from swap fees, which makes it very sustainable as there is no sell pressure on farming tokens.
Dapp: https://app.acryptos.com/aclm/?g=chain&a=optimism

Some stats:
- [26 Vaults](https://app.acryptos.com/vaults/?g=chain&a=optimism) launched on Optimism
- Yields and TVL for single-token vaults on Optimism (https://defillama.com/yields?project=acryptos)
- Aside from BSC (our native chain), Optimism Chain is our top TVL chain right now
- Some of our ACLM vaults have great performance metrics. Eg. we've been hosting
90%+ of active liquidity on UniV3 for USDT/DAI, USDC/USDT
20%+ on wstETH/ETH

Sustainable APY via the ACLM vaults on UniswapV3 is really the next step forward for liquidity on defi.
Feel free to check out our vaults, and drop in any questions you have about the strategies involved!

---

We've submitted for RetroPGF3 as well. Please do check it out, and thanks for the support!
https://vote.optimism.io/retropgf/3/application/0x5cc8dc85d40909999a55201f90c714b334cc502f013209497a271934e604b9d0

**Comment by sbvegan** [comment]
Very cool, thanks for sharing what you're building :)

**Reply by CryptoBKT** [reply]
Thanks! Let us know if you managed to check out the dapp 😉
